### ISSUE TITLE
Assignment of unique handles in 1.1.82 SDK

### ISSUE THREAD

**Issue by lawrencem99** [KhronosGroup/Vulkan-Hpp#266]
I am wondering if anybody else is getting compiler errors when assigning to a unique handle,

error: non-static reference member ‘const vk::DispatchLoaderStatic& vk::ObjectDestroy<vk::Device, vk::DispatchLoaderStatic>::m_dispatch’, can’t use default assignment operator

**Comment by asuessenbach** [comment]
Could you please provide some source that provokes that error?

**Comment by sguimmara** [comment]
Yes i have it too (on Windows, using C++ 17 and MSVC, SDK 1.1.82). It triggers when I try to assign a UniqueHandle that is in a struct 

    struct SwapchainImage
    {
        vk::UniqueFramebuffer framebuffer;
    }

later...

    SwapchainImage image;

    image.framebuffer = device->createFramebufferUnique(info);

**Comment by sguimmara** [comment]
After updating from 1.1.82 to 1.1.85, the problem seems to have disappeared. Did you make modification to move semantics ?

**Comment by lawrencem99** [comment]
Thank you sg I forgot about this.

Your experience is the same as mine.

The problem with having broken releases is that often package maintainers won't immediately package newer versions of the SDK eg. Fedora's vulkan-headers package has been on .82 for two months. So then I have to setup the newer SDK manually which shouldn't really be necessary.
